packages/react-vapor/src/components/table-hoc/TableRowConnected.tsx

```diff
@@ -190,9 +190,13 @@ class TableRowConnected extends React.PureComponent<
     };
 
     private handleDoubleClick = () => {
-        _(this.props.actions)
-            .filter((action: IActionOptions) => action.callOnDoubleClick)
-            .forEach((action: IActionOptions) => action.trigger());
+        _(this.props.actions).each((action: IActionOptions) => {
+            if (action.callOnDoubleClick && action.link) {

```

Comment:
🤔 

Why not 

```
 .filter((action: IActionOptions) => action.callOnDoubleClick))
 .forEach((action: IActionOptions) => {
    if (action.link) { 
       window.location.href = action.link;
     } else if (action.trigger) {
       action.trigger();
     }
 });
```

Reply:
The solution proposed by @dmgauthier is better performance wise as it iterates only one time over all actions whereas your solution iterates one time over all actions + one time over double click actions. Practically thought, it makes no real difference as we won't have enough action to make a significant difference. I would go with readability here.